frontends/web/src/routes/device/bitbox02/passphrase.tsx

```diff
@@ -27,12 +27,6 @@ import { View, ViewButtons, ViewContent, ViewHeader } from '../../../components/
 import { PointToBitBox02 } from '../../../components/icon';
 import { Status } from '../../../components/status/status';
 
-// enabling has 6 dialogs with information
-const INFO_STEPS_ENABLE = 5;
```

Comment:
Nit: We had this number in one place with a comment what it is all about.

Now this is in 4 different places in the component without any comment (or I missed it somehow), do you prefer having it this way or would you mind adding something similar back with a useful comment?

Reply:
imo. the comment(s) were necessary because the logic was so complicated/intertwined. now it is so straight forward that a developer working on this code will understand what is happening without comment imo.
- the  infoStep is only relevant for the enable wizard now
- the info step starts at 0 , if you press back it means you abort
- when the infoStep reaches 5 it will set the passphrase on continue because its the last step

this is all really obvious from the code and does not need comments imo. but let me know if you disagree, ofc. I can add some.

Reply:
I agree, however I don't think it is ideal to have the number 5 hardcoded in 4 places. Think about if we need to add or remove one more pages.

but just a nit. It is ok for me

Reply:
totally agree, should I add a `const FINAL_INFO_STEP = 5` ?